### DISCUSSION TITLE
Adding RTL support

### DISCUSSION THREAD

**Discussion by koossaayy** [spatie/laravel-support-bubble#10] [Ideas]
Hey
I managed to add RTL support, tweaking some styles and some JavaScript, but the thing is, it's working only if the direction attribute is present in the html 

`<html lang="ar-TN" dir="rtl">
` 

But the problem is when the direction attribute is missing, the styles are not showing up correctly. Do I submit a PR using only those ? or try to see if I can do some JavaScript again ?

**Comment by freekmurze** [comment]
Feel free to create a PR for this 👍

**Reply by AlexVanderbist** [reply]
PR has been merged!
